packages/embeds/embed-core/src/embed.test.ts

```diff
@@ -31,15 +31,17 @@ describe("Cal", () => {
     CalClass = (await import("./embed")).Cal;
   });
 
-  beforeEach(() => {
-    cal = new CalClass("test-namespace", []);
-    window.Cal.config = { forwardQueryParams: true };
-    // Mock the getConfig method
-    cal.getConfig = vi.fn().mockReturnValue({ calOrigin: "https://app.cal.com" });
-  });
+
 
   describe("createIframe", () => {
-    describe("params handling", () => {
+    describe("params handling with forwardQueryParams feature enabled", () => {
+      beforeEach(() => {
+        cal = new CalClass("test-namespace", []);
+        window.Cal.config = { forwardQueryParams: true };
+        // Mock the getConfig method
+        cal.getConfig = vi.fn().mockReturnValue({ calOrigin: "https://app.cal.com" });
```

Comment:
Moved here to convey that all tests within the describe use forwardQueryParams feature